### PR TITLE
Allow more combinations in variables

### PR DESCRIPTION
**Pull request by mati-k**

### CODE REVIEW

File: triggers.cwt [hunk 1]
```diff
@@ -2615,7 +2615,9 @@ alias[trigger:is_vassal] = bool
 ## scope = any
 ###Returns true if the specified variable is equal to X.
 alias[trigger:is_variable_equal] = {
+	## cardinality = 1..2
```
[the remark this answers is not in the document]
Comment (maxice8) [reply triggers.cwt:2618]: I think it is better it be split into 2, of the top of my head I think this works.

```
## scope = any
###Returns true if the specified variable is equal to X.
alias[trigger:is_variable_equal] = {
	which = value[variable]
	value = float
}

## scope = any
###Returns true if the specified variable is equal to X.
alias[trigger:is_variable_equal] = {
	## cardinality = 2..2
	which = value[variable]
}
```

[the remark this answers is not in the document]
Comment (mati-k) [reply triggers.cwt:2618]: Sure will do that tomorrow, want me to apply that to other variable stuff too?

	## cardinality = 0..inf
	change_variable = {
		## cardinality = 1..2
		which = value[variable]
		## cardinality = 0..1
		value = float
	}

for example that is current for change_variable, i don't think that'd be allowed in engine

[the remark this answers is not in the document]
Comment (maxice8) [reply triggers.cwt:2618]: If the same problems apply as is_variable_equal then please do, thanks.

[the remark this answers is not in the document]
Comment (mati-k) [reply triggers.cwt]: Okay pushed

**Pull request merged**